CONTRIBUTING.md

```diff
@@ -194,53 +194,22 @@ to do so. Travis also runs shellcheck.
 
 ## Releasing
 
-To cut a release, push a new tag (versioning discussed below).
-
-### Tagging a release
-
-1. Run `make changes` to review the changes since the last
-   release. Based on the changes, decide what kind of release you are
-   doing (bugfix, feature or breaking).
+1. Use [github-changelog-generator](https://github.com/digitalocean/github-changelog-generator) 
+   to list the changes since the last release 
+   and decide what kind of release you are doing (bugfix, feature or breaking).
    `doctl` follows [semantic versioning](https://semver.org), ask if you aren't sure.
 
-1. Tag the release using `BUMP=(bugfix|feature|breaking) make tag`.
-   (Bugfix, feature and breaking are aliases for semver's patch, minor and major.
-   BUMP will also accept `patch`, `minor` and `major`, if you prefer). The command
-   assumes you have a remote repository named `origin` pointing to this
-   repository. If you'd prefer to specify a different remote repository, you can
-   do so by setting `ORIGIN=(preferred remote name)`.
-
-The new tag triggers the release.
-
-### Snap
```

Comment:
This section is still important for building a snap package locally for testing. Rather than removing completely, let's figure out a place for it in the doc. Or maybe even move it to a README in the `snap/` directory? :thinking: 

https://github.com/digitalocean/doctl/tree/main/snap

Reply:
Sounds good. Will update today. Thanks for the review :+1: